components/automate-cli/cmd/chef-automate/automateHADeployExistingInfra.go

```diff
@@ -203,7 +203,42 @@ func (e *existingInfra) populateCertificateTomlFile() error {
 	e.log.Debug("Certificate TOML written to %s\n", CERTIFICATE_TEMPLATE_TOML_FILE)
 	return nil
 }
+
+type DefaultBackendCerts struct {
+	RootCA    string `toml:"rootCA"`
+	AdminCert string `toml:"admin_cert"`
+	AdminKey  string `toml:"admin_key"`
+	SslCert   string `toml:"ssl_cert"`
+	SslKey    string `toml:"ssl_key"`
+}
+
 func (e *existingInfra) addDNTocertConfig() error {
+	e.log.Debug("custom certificate enabled status", e.config.Opensearch.Config.EnableCustomCerts)
+	if !e.config.Opensearch.Config.EnableCustomCerts {
```

Comment:
we can keep condition for pg seperately. 
because any one can choose the custom cert for either of 1

Reply:
done